setup.py

```diff
@@ -10,8 +10,9 @@
     description="An asynchronous Tasks Library using asyncio",
     packages=['pytask_io'],
     install_requires=[
-        'redis',
-        'dill'
+        'redis>=3.3.11,<4.0.0',
+        'dill>=0.3.1.1,<0.4.0.0',
+        'toolz>=0.10.0,<0.11.0',     
```

Comment:
We can remove toolz now

Reply:
https://github.com/joegasewicz/pytask-io/pull/36

Reply:
Also @MakuZo I have updated the https://github.com/joegasewicz/pytask-io/blob/master/CHANGELOG.md in my pr:
https://github.com/joegasewicz/pytask-io/pull/36/files
If you want to also add your changes / issue it would be great, thank you

Reply:
btw  we now have a gitter at https://gitter.im/pytaskio/community

Reply:
@joegasewicz 
I've removed toolz from setup.py

Reply:
Do you want me to update the changelog?